CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Export
 Service to export YourLoops data in selected formats, either csv, json or xls.
 
+## 1.8.8 - 2024-09-10
+### Engineering
+- Security fix
+- 
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
**🤓 nitpick:** ‏Additional "-" to remove?

```suggestion

```